packages/generator/test/renderer.test.ts

```diff
@@ -98,13 +97,23 @@ describe("Rendering Tests", () => {
     await Promise.all(
       _.keysIn(apiConfig).map(async apiGroup => {
         allApis[apiGroup] = await Promise.all(
-          processApiFamily(apiGroup, apiConfig, apiInputDir)
+          renderer.processApiFamily(apiGroup, apiConfig, apiInputDir)
         );
       })
     );
     const list = renderer.renderOperationList(allApis);
     expect(list).to.be.a("string");
   });
+
+  it("Throws error when id is missing", async () => {
+    // expect().to.be.rejected was not working here
```

Comment:
just curious..is this because the function returns  array of unresolved promises?

Reply:
It's not working because `chai-as-promised` is loaded as a plugin.

Reply:
nope, wasn't working with chai-as-promised either. and yes, it does return the array, but no matter what I tried, it wasn't working properly :/